addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -444,16 +447,22 @@ export class GraphRenderer extends Component {
             scaleLabel: {
                 display: Boolean(groupBy.length),
                 labelString: groupBy.length ? fields[groupBy[0].fieldName].string : "",
+                fontColor: this.cookies.current.color_scheme === "dark" ? getColor(15, this.cookies.current.color_scheme) : null,
```

Comment:
those lines are too long. Could you enable the prettier/eslint tooling? `./addons/web/tooling/enable.sh`. It automatically prettifies/lints altered files when you commit (pre-commit hook). Thanks

Reply:
I have done the necessary changes. Thank You.